.github/workflows/publish-package-docs.yaml

```diff
@@ -66,23 +66,23 @@ jobs:
           python -m pip install -r docs/docsite/requirements.txt
           popd
 
-      - name: Set the COLLECTION_LIST variable
+      - name: Set collection list
```

Comment:
Nit: it's usually best to separate refactoring into a dedicated commit/PR so the only changes left are related to the PR title directly.

Reply:
ack. given this is just for a demo I didn't think it would be strictly necessary. I take your point though.